model/pail_storage.go

```diff
@@ -52,19 +52,6 @@ func (t PailType) Create(ctx context.Context, env cedar.Environment, bucket, pre
 		if err != nil {
 			return nil, errors.WithStack(err)
 		}
-	case PailLegacyGridFS, PailGridFS:
-		client := env.GetClient()
-		conf := env.GetConfig()
-
-		opts := pail.GridFSOptions{
-			Database: conf.DatabaseName,
-			Name:     bucket,
-			Prefix:   prefix,
-		}
-		b, err = pail.NewGridFSBucketWithClient(ctx, client, opts)
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
```

Comment:
Why is this being removed?

Reply:
Because we removed GridFS from pail, so this breaks the build if you use an updated version.

Reply:
Ah, I get it.
Should we remove [these](https://github.com/evergreen-ci/cedar/blob/8e0bf9facd32bfe09460d0b84f5801204bdf87be/model/pail_storage.go#L21-L22) too?